source/Ship.cpp

```diff
@@ -3499,6 +3499,12 @@ bool Ship::Carry(const shared_ptr<Ship> &ship)
 	// Check only for the category that we are interested in.
 	const string &category = ship->attributes.Category();
 
+	// NPC ships should always transfer cargo.
+	bool shouldTransferCargo = true;
+	// Player ships should only transfer cargo if they set the AI preference.
+	if(IsYours())
+		shouldTransferCargo = Preferences::Has("Fighters transfer cargo");
```

Comment:
You can write all of this in one line and make the variable const as well.

```c++
const bool shouldTransferCargo = !isYours || Preference::Has("Fighters transfer cargo";
```

Reply:
I'll make this change

Reply:
Quick question aside from the change.  Why choose const?  Is it because it won't change from this point onward?

Reply:
Yes.